.github/workflows/cd.yml

```diff
@@ -0,0 +1,80 @@
+# Un problème est le manque de priorité entre les
+# workflows, il n'est actuellement pas possible de donner
+# un ordre d'exécution, l'idéal serait d'exécuter CI avant CD
```

Comment:
Thank you for commenting on this part, this will allow us to come back to it after finding a solution

Reply:
I hope they will add a way to do it